src/app/components/auth/auth.component.scss

```diff
@@ -1,27 +1,31 @@
-.auth-content{
-    display: flex;
+.auth-content {
+  display: flex;
 
-    height: 200px;
-    width: 400px;
-    margin: auto;
-    background: rgba(240,240,240,0.93);
-    box-shadow: 0 2px 12px grey;
-    position: relative;
+  height: 200px;
+  width: 400px;
+  margin: auto;
+  background: rgba(240, 240, 240, 0.93);
+  box-shadow: 0 2px 12px grey;
+  position: relative;
 }
 
-form{
-    flex: 1;
-    display: flex;
-    flex-direction: column;
+form {
+  flex: 1;
+  display: flex;
+  flex-direction: column;
 }
 
-.auth-inputs{
-    padding: 20px;
-    padding-top: 25px;
+.auth-inputs {
+  padding: 20px;
+  padding-top: 25px;
 }
 
-.auth-actions{
-    justify-content: space-around;
-    margin-top: auto;
-    margin-bottom: 15px;
-}
\ No newline at end of file
+.auth-actions {
+  justify-content: space-around;
+  margin-top: auto;
+  margin-bottom: 15px;
+}
+
+button[type="submit"] {
+  float: right;
```

Comment:
Float is best used when the content is surrounded with text. Consider using something like flexboxing, relative positionning, or other instead of float.